### PR TITLE
RN 21.19

### PR DESCRIPTION
**Pull request by khanzadyan**

### CODE REVIEW

File: content/_releases/2021-05-13.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+*   Set the new environment variable `DEBUG_TASK_TIMEOUT` to control the time.
+*   If the variable is not set, platform will use the default value set in `FORCE_DESTROY_DEBUG_TASK_TIMEOUT_SEC` variable.
+*   If multiple `DEBUG_TASK_TIMEOUT` variables are set for the flow components (i.e. for mapper and component after), the system will take bigger value of them.
+
+
+## Improvements and Updates
+
+### Component version display
+
+You can now set you own component version to show in the platform UI when you use
```
Comment (luzhok) [inline content/_releases/2021-05-13.md:36]: "set youR own..."

File: content/_releases/2021-05-13.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+the `version` parameter without letter **v** in you `component.json` configuration.
```
Comment (luzhok) [inline content/_releases/2021-05-13.md:37]: "in youR component.json"

File: content/_releases/2021-05-13.md (hunk 1 above)
Comment (khanzadyan) [reply content/_releases/2021-05-13.md]: Thank you for spotting. Corrected.

File: content/_releases/2021-05-13.md (hunk 2 above)
Comment (khanzadyan) [reply content/_releases/2021-05-13.md]: corrected

**Pull request merged**